src/ReTestItems.jl

```diff
@@ -245,7 +258,7 @@ function _runtests(shouldrun, paths, nworkers::Int, nworker_threads::String, wor
         if is_running_test_runtests_jl(proj_file)
             # Assume this is `Pkg.test`, so test env already active.
             @debugv 2 "Running in current environment `$(Base.active_project())`"
-            return _runtests_in_current_env(shouldrun, paths, proj_file, nworkers, nworker_threads, worker_init_expr, testitem_timeout, retries, verbose_results, debug, report, logs)
+            return _runtests_in_current_env(shouldrun, paths, proj_file, nworkers, nworker_threads, worker_init_expr, test_end_expr, testitem_timeout, retries, verbose_results, debug, report, logs)
         else
```

Comment:
👀 any idea why this line isn't covered?

Reply:
maybe fixed by latest commit -- but i think this is just CodeCov being confusing

this line is exercised whenever a package using ReTestItems.jl has test triggered via `Pkg.test()`, so it is covered by existing tests for our existing `test/packages/`, but in the latest commit i add the new TestEndExpr.jl package as well to make sure that works with `Pkg.test()` too